cmd/cdi-operator/operator.go

```diff
@@ -84,7 +85,11 @@ func main() {
 	}
 
 	managerOpts := manager.Options{
-		Namespace:                  namespace,
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{
+				namespace: {},
```

Comment:
No need to include `namespace` anywhere?

Reply:
wdym? it's included `namespace: {},`

Reply:
I mean the namespace we get above (`namespace := util.GetNamespace()`). Just wondering if we don't have to assign it somewhere as we did before.

Reply:
it's included as the key in `map[string]cache.Config`